tests/hook_main.py

```diff
@@ -3593,6 +3593,7 @@ def custom_placeholder_sub_topic_get_detail(transaction):
         db.session.add(event_sub_topic)
 
         custom_placeholder = CustomPlaceholderFactory(event_sub_topic_id=1)
+        custom_placeholder.event_sub_topic_id = 1
```

Comment:
Why?

Reply:
@iamareebjamal Oh, sorry. This was for testing when I was not able to figure out the exact reason for tests failing. I will undo this.